docs/danews.md

```diff
@@ -0,0 +1,147 @@
+Nielbo, K. L., Baglini, R. B., Vahlstrup, P. B., Enevoldsen, K. C., Bechmann, A., & Roepstorff, A. (2021, January). News information decoupling: An information signature of catastrophes in legacy news media. https://eadh2020-2021.org/
+
+**Is there a repository that links to any or all papers or systems that use the dataset?**
+No.
+
+**What (other) tasks could the dataset be used for?**
+The scale of the dataset makes it suitable for NLP tasks such as language modelling. Similarly, the structure
+of the articles make it a suitable dataset for training text-summerization models.
+
+**Is there anything about the composition of the dataset or the way it was collected and preprocessed/cleaned/labeled that might impact future uses?**
+
+This dataset is static and thus does not evolve over time with the language, thus will become increasingly outdated over time.
+
+
+**Are there tasks for which the dataset should not be used?**
+This dataset contains Danish articles and thus should not be used for Danish language task.
+As the writers of the content is predominantly journalists, it reflect a certain social group which is unlikely to reflect Danish language as a whole.
+
+## Distribution
+**Will the dataset be distributed to third parties outside of the entity (e.g., company, institution, organization) on behalf of which the dataset was created?**
+No.
+
+
+# Post processing
+DaNews have been filtered for quality using a series of heuristic filters and removing repitious texts. 
+Following the filtering DaNews is deduplicated to remove exact and near-duplicates.
+
+A total of 2338728 (0.09%) were considered low quality and a 
+total of 1048815 (0.04%) documents were considered duplicates.
+
+## Quality Filter:
+For the quality filtering DaNews applies a filter akin to [2] which filters text which does not:
+
+- Contain at least 2 Danish stopwords. For the stopword list we use the one used in SpaCy v.3.1.4.
+- Have mean word length between 3-10.
+- Have a token length between 50-100 000.
+- Have a less than 5 000 000 characters.
+- Have a less than 60% words containing an alphabetic character.
+- Have a symbol to word ratio lower than 10% for hashtags an ellipsis.
+- Have less than 90% of lines starting with a bullet point.
+- have less than 30% of lines ending with an ellipsis.
+
+- Have low high degree of repitiuous text:
+  - Less than 20% duplicate lines.
+  - Less than 20% duplicate paragraphs.
+  - Where the top 2-4 grams constitute less than 20%, 18%, 16%, respectively, of the text. 
+  - Where the duplicate 5-10 grams constitute less than 25%, 24%, 23%, 22%, 21%, 20% of the text, respectively.
+
+## Deduplication
+The deduplication removed all documents with a 13-gram similarity higher than 80% following the MinHash algorithm [1] using 128 permutation.
```

Comment:
Did we end up using 128? I thought this was reduced even further?
```suggestion
The deduplication removed all documents with a 13-gram similarity higher than 80% following the MinHash algorithm [1] using 128 permutations.
```

Reply:
corrected